src/views/wallet/WalletView.tsx

```diff
@@ -34,15 +34,23 @@ const WalletView: React.FC = (): JSX.Element => {
   // we have to add 'undefined'  as default value
   const keystore = useObservableState(keystoreService.keystore$, undefined)
 
+  const reloadButton = useMemo(
+    () => (
+      <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
+        <Styled.RefreshButton typevalue={'outline'} onClick={reloadBalances}>
```

Comment:
[nitpick] We don't want an outline for the button anymore.

Reply:
@veado i left it for the sake of saving another styles. background and border reset at the `RefresButton` styles

Reply:
Cool, thx!